java/java-binding/src/main/java/com/risingwave/java/binding/Binding.java

```diff
@@ -82,4 +82,6 @@ public class Binding {
     static native long streamChunkIteratorNext(long pointer);
 
     static native void streamChunkIteratorClose(long pointer);
+
+    static native long streamChunkIteratorGenerate(String str);
```

Comment:
Better to rename it to `streamChunkIteratorFromPretty` to be consistent with the name in rust.